.github/workflows/fronts.yml

```diff
@@ -0,0 +1,34 @@
+name: DCR Fronts Ophan tag status
+on:
+  # Triggers the workflow on push and pull request events but only for the main branch
+  push:
+    branches: [main]
+
+  # Allows you to run this workflow manually from the Actions tab.
+  workflow_dispatch:
+
+jobs:
+  data-link-name:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Setup deno
+        uses: denolib/setup-deno@v2
+        with:
+          deno-version: v1.21.0
+
+      - name: Run Deno script
+        id: text
+        run: |
+          echo "About to run Deno script"
+          TEXT=$(deno run --allow-net https://gist.githubusercontent.com/mxdvl/75007429b33d14ff2838c3cb51b50673/raw/73677246f9ea1652a516e8b16bed4ce842b084d0/data-link-name.ts)
+          echo "::set-output text=$TEXT"
+
+      - name: Create or Update Comment
+        uses: peter-evans/create-or-update-comment@v2.0.0
+        with:
+          comment-id: 557858210
```

Comment:
I think you might want to use something from this doc:
https://github.com/peter-evans/create-or-update-comment#where-to-find-the-id-of-a-comment

Reply:
I think these two are right:
- `data-link-name`: https://github.com/guardian/dotcom-rendering/issues/4692#issue-1211151898
- `data-component`: https://github.com/guardian/dotcom-rendering/issues/4698#issue-1212191944

Reply:
Actually no, but I’ve opted for doing the updating in Deno directly with Octokit.